src/components/Carousel.js

```diff
@@ -381,6 +384,11 @@ module.exports = React.createClass({
         );
     },
 
+    getInitialImage () {
+        var images = ReactDOM.findDOMNode(this.item0).getElementsByTagName('img');
+        return images && images[this.props.selectedItem];
```

Comment:
Just realised that there is a bug in the existent code. This will only return the initial image if `selectedItem` is `0`.

We need to change this to be: 

```javascript
var images = ReactDOM.findDOMNode(this['item' + this.props.selectedItem]).getElementsByTagName('img');
return images && images[0];
```